### PR TITLE
Modify project.clj to be lein 2.8 compatible

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**
- Cleanup dockerfiles, and add `printenv` for build verbosity
- Enable HTTP traffic in project as a dependency of `org.apache.pdfbox/pdfbox` is available through HTTP only 
- Set HTTPS mirror from apache to force HTTPS when possible

**Connected issues**
Fixes part of #8

### CODE REVIEW

File: docker/bootstrap/Dockerfile [hunk 1]
```diff
@@ -2,15 +2,16 @@ FROM openjdk:8-stretch
 MAINTAINER daemonsthere@gmail.com
 
 ENV LEIN_ROOT 1
-ENV LEIN_VERSION="2.7.1"
+ENV LEIN_VERSION="2.8.3"
```
Comment (MeanderingCode) [inline docker/bootstrap/Dockerfile]: Now that we're compatible and running 2.8, we should simply build `FROM clojure/openjdk8-lein-2.8.3`

File: docker/orcpub/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,8 @@ ARG BRANCH="develop"
 
 # Get orcpub source
 WORKDIR /opt
-RUN git clone --branch ${BRANCH} https://github.com/${REPO}/orcpub.git &&\
+RUN printenv &&\
+	git clone --branch ${BRANCH} https://github.com/${REPO}/orcpub.git &&\
```
Comment (MeanderingCode) [inline docker/orcpub/Dockerfile]: I assume this was a hack to auto-build on DockerHub, but still pull from this repo.

This should just copy in from the context directory.

Comment (MeanderingCode) [reply docker/orcpub/Dockerfile]: Additionally, I'd like to see the the runner build `FROM openjdk/8-jre-alpine`, for image size.

File: project.clj [hunk 3]
```diff
@@ -1,3 +1,8 @@
+; Allow http connection, as org.apache.pdfbox/pdfbox has http dependnecies
+(require 'cemerick.pomegranate.aether)
+(cemerick.pomegranate.aether/register-wagon-factory!
+ "http" #(org.apache.maven.wagon.providers.http.HttpWagon.))
```
Comment (MeanderingCode) [inline project.clj:4]: Did you determine that we need the dependency that is coming from an insecure repository? If not, we can exclude it explicitly.

Comment (MeanderingCode) [reply project.clj:4]: See https://worace.works/2018/05/29/identify-non-https-transitive-leiningen-dependencies/

File: project.clj [hunk 4]
```diff
@@ -160,8 +166,8 @@
   ;; Please see:
   ;; https://github.com/bhauman/lein-figwheel/wiki/Using-the-Figwheel-REPL-within-NRepl
 
-  :uberjar-inclusions [".ebextensions"]
-  :jar-inclusions [".ebextensions"]
+  :uberjar-inclusions [#"^\.ebextensions"]
+  :jar-inclusions [#"^\.ebextensions"]
```
Comment (MeanderingCode) [inline project.clj:170]: This is a good find! I did not see it when I was looking for improper regexs for the offending `ClassCastException`.
Out of curiosity, how did you identify this as the issue?

**Review by MeanderingCode — CHANGES_REQUESTED** [review]
This is great! I'm thrilled we can come up to date.
Just a couple requests, too keep you busy :)

File: docker/orcpub/Dockerfile (hunk 2 above)
Comment (Demonsthere) [reply docker/orcpub/Dockerfile]: Yes, this is kinda a hack, as the project files are outside of the default docker context, which starts where the Dockerfile is. To have it use the files in the repo without cloning stuff again, we would need to set docker context to `/` and modify all `COPY/ADD` to reflect that the context has changed.

File: project.clj (hunk 4 above)
Comment (Demonsthere) [reply project.clj:170]: found it here: https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L418

File: project.clj (hunk 3 above)
Comment (Demonsthere) [reply project.clj:4]: I don't think this is what we are looking for. 
> If they turn out to be things you actually need, you may have to go find a compatible version from an HTTPS source to replace them with

The exclusions means ignore this dependency, and not ignore that this is http :<

File: docker/bootstrap/Dockerfile (hunk 1 above)
Comment (Demonsthere) [reply docker/bootstrap/Dockerfile]: Image changed

File: project.clj (hunk 4 above)
Comment (MeanderingCode) [reply project.clj:170]: Just by doing a close compare? I was trying to find it via stacktraces, then just looking for all clojure functions that have regex args. Didn't notice this, myself.

Glad you found it :)

File: docker/orcpub/Dockerfile (hunk 2 above)
Comment (MeanderingCode) [reply docker/orcpub/Dockerfile]: I don't know what's possible on Docker Hub, but in a shell you can execute `docker build -f docker/bootstrap/Dockerfile .` from the project root and the project root will be your context directory.

File: project.clj (hunk 3 above)
Comment (MeanderingCode) [reply project.clj:4]: Yes, if we need that dependency, let's keep this global and file an issue in our repo and on their issue tracker.

File: docker/orcpub/Dockerfile (hunk 2 above)
Comment (MeanderingCode) [reply docker/orcpub/Dockerfile]: Looks like you changed this. We'll see what we can do with Hub.

File: project.clj (hunk 3 above)
Comment (MeanderingCode) [reply project.clj:4]: Opened issue #27.
Will merge this now.

**Review by MeanderingCode — APPROVED** [review]
This is excellent!

**Pull request merged**